lms/djangoapps/gdpr_user_retirement/tests/test_views.py

```diff
@@ -0,0 +1,84 @@
+"""
+Test cases for GDPR User Retirement Views
+"""
+from django.urls import reverse
+from rest_framework.test import APIClient, APITestCase
+from openedx.core.djangoapps.user_api.models import RetirementState, UserRetirementStatus
+from student.tests.factories import UserFactory
```

Comment:
@xitij2000, `pytest` can't collect these tests.
```suggestion
from common.djangoapps.student.tests.factories import UserFactory
```